src/components/TicketValidationMessage/TicketValidationMessage.jsx

```diff
@@ -28,6 +34,12 @@ const _TicketValidationMessage = ({
           {` `}para continuar.
         </>
       );
+    } else if (showValidations && isDniDuplicated) {
+      message = `Necesitamos que completes el ticket con valores válidos. El DNI ingresado ya
```

Comment:
What about something like: 

_Parece que ya tenemos una entrada registrada con este número de documento. ¿Perdiste tu entrada? ¡Contactanos por nuestras redes sociales y te ayudaremos!_

(Same message style for the next error).

Reply:
This message is for when the user inputted the same DNI when buying multiple tickets; we're not handling the error for duplicated DNI's coming from the backend yet.

Reply:
@joelalejandro @asermax You guys decide and let me know if I should change it.